docs: jcloud docs revamp for instance (#5759)
jina-ai/jina: docs/concepts/jcloud/index.md

#### Project folder

````{tip}

> The best practice of creating a Jina AI Cloud project is to use:

Reviewer: ```suggestion
The best practice for creating a Jina AI Cloud project is to use:
```

```bash
jc new
```

> This ensures the correct project structure accepted by Jina AI Cloud.

Reviewer: ```suggestion
This ensures the correct project structure that is accepted by Jina AI Cloud.
```

:width: 70%
```

## Configurations

> Please refer to {ref}`Configurations <jcloud-configurations>` for configuring the Flow on Jina AI Cloud.

Author: fixed!

## Restrictions

> Jina AI Cloud scales according to your needs. You can demand different instance type with GPU/Memory/CPU pre-defined based on the needs of your Flows and Executors. If you have specific resource requirements, please contact us [on Slack](https://jina.ai/slack) or raise a [GitHub issue](https://github.com/jina-ai/jcloud/issues/new/choose).

Reviewer: ```suggestion
Jina AI Cloud scales according to your needs. You can demand different instance types with GPU/memory/CPU predefined based on the needs of your Flows and Executors. If you have specific resource requirements, please contact us [on Slack](https://jina.ai/slack) or raise a [GitHub issue](https://github.com/jina-ai/jcloud/issues/new/choose).
```
